config/default/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
I think we'll want to keep `IfNotPresent` in the repo, but I'm guessing this is just for development?